docs/development/building/Building-in-Ubuntu.md

```diff
@@ -1,26 +1,34 @@
+---
+sidebar_position: 1
+sidebar_label: Building in Ubuntu
+title: Building in Ubuntu
+---
+
 # Building in Ubuntu
 
 Building for Ubuntu platform is remarkably easy.
 This document is tested and based on Ubuntu 24.04 LTS release and can also be used for WSL.
 
 ### Clone Betaflight Repository and Install Toolchain
 
+Cloning the Betaflight repository will create a folder named `betaflight` in your current folder and download a copy for local use. You may wish to begin in a project folder such as `~/Git/`, `~/Github/`, or `~/My-Projects/`. Your local clone will not be automatically synced when Betaflight's github repository is updated; Syncing must be done manually. If development and pull-requests is intended, then please see the "[Git and Github](../git)" section. The following command-lines will install necessary prerequisites, clone and setup Betaflight.
+
 ```
-$ sudo apt update && sudo apt upgrade
-$ sudo apt install build-essential libblocksruntime-dev libtool git curl clang-18
-$ git clone https://github.com/betaflight/betaflight.git
-$ cd betaflight
-$ make arm_sdk_install
-$ make configs
+sudo apt update && sudo apt upgrade
+sudo apt -y install build-essential git curl clang-18 python3 python-is-python3
```

Comment:
Where is python used in configurator :zany_face: 

- Removing `libblocksruntime-dev` and `libtool` - were probably needed for NWjs implementation ?
- Do we need them again when we agree on a native build wrapper ?

Reply:
These were/are in the Firmware section, not Configurator section.

Reply:
```
Package: libtool
Description: Generic library support script
 This is GNU libtool, a generic library support script.  Libtool hides the complexity of generating special library
 types (such as shared libraries) behind a consistent interface.  To use libtool, add the new generic library building
 commands to your Makefile, Makefile.in, or Makefile.am.  See the documentation for details.  Libtool supports building
 static libraries on all platforms.
```

```
Package: libblocksruntime-dev
Description: Blocks Runtime (development files)
 This package contains development headers for building software that uses blocks.

 Blocks are a proposed extension to the C, Objective C, and C++ languages developed by Apple to support the Grand
 Central Dispatch concurrency engine.
```

Reply:
```
$ grep Iirn python ./src/
./src/main/target/serial_post.h:28:> python ./src/utils/gen-serial-j2.py > ./src/main/target/serial_post.h
./src/utils/templates/serial_post.h:11:> python ./src/utils/gen-serial-j2.py > ./src/main/target/serial_post.h
./src/utils/templates/serial_post.h:29:{# TODO - maybe store configuration in this file, not Python one #}
./src/utils/make-build-info.py:1:#!/usr/bin/env python3
./src/utils/dfuse-pack.py:1:#!/usr/bin/python
```

Reply:
- any further ideas of the two packages considering the above Descriptions?
- i am able to compile single target and `all` and `all_configs` without error.

Reply:
solved: found both of those questioned packages are already installed. maybe after build-essentials. unclear without reverting VM snapshot.